koji_fedoramessaging_messages/rpm.py

```diff
@@ -223,20 +224,27 @@ def owner(self):
 
     @property
     def instance(self) -> str:
-        return self.body["instance"]
+        return self.body.get("instance")
 
     @property
     def sigkey(self) -> str:
-        return self.body["sigkey"]
+        return self.body.get("sigkey")
 
     @property
     def rpm(self) -> dict:
-        return self.body["rpm"]
+        return self.body.get("rpm")
 
     @property
     def build(self) -> dict:
-        return self.body["build"]
+        return self.body.get("build")
 
     @property
     def sighash(self) -> str:
-        return self.body["sighash"]
+        return self.body.get("sighash")
+
+    @property
+    def packages(self) -> List[str]:
+        try:
+            return [self["build"]["package_name"]]
+        except (KeyError, TypeError):
+            return []
```

Comment:
By reading the field description, I think it makes sense, but I'm not sure if it's always a package or if there can be modules/flatpaks/containers in there.

The `TypeError` is to handle the case where `self["build"]` is `None`.